docs/docs/users/filecoin_services.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Filecoin Services
+hide_title: true
+sidebar_position: 6
+---
+
+<p align="center">
```

Comment:
```suggestion
<p align="center" style="display: flex; justify-content: space-around;">
```

Reply:
May not work 100%, but should help evenly distribute the images

This is what I currently see without the change:
<img width="959" alt="Screenshot 2025-01-24 at 9 45 45 AM" src="https://github.com/user-attachments/assets/0d65eff7-1265-4d14-9444-bc5454a7fd10" />

Reply:
I like it. FYI the suggested change doesn't work out of the box.

```
 The `style` prop expects a mapping from style properties to values, not a string. For example, style={{marginRight: spacing + 'em'}} when using JSX.
```